app/index.js

```diff
@@ -23,7 +23,7 @@ module.exports = generators.Base.extend({
                 type: 'input',
                 name: 'name',
                 message: 'Your project name',
-                default: this.appname // Default to current folder name
+                default: 'react-engine-generator' // Default to current folder name
```

Comment:
If someone has already created the folder they want the project in and is running the generator in that folder, this will result in another folder named react-engine-generator created under it.